### DISCUSSION TITLE
Don't know where to get started.

### DISCUSSION THREAD

**Discussion by notramo** [helix-editor/helix#7366] [General]
When Helix is opened empty, the user is presented with a blank screen, instead of e.g. a cheatsheet.

**Comment by kirawi** [comment]
Try to begin with `hx --tutor` as well as the docs: https://docs.helix-editor.com/

**Reply by notramo** [reply]
At least the app should display a text like this:

| action | command |
| ---- | ---- |
|Tutorial| `:tutor`|
|Quit|`:q`|
See https://docs.helix-editor.com for documentation.

**Reply by kirawi** [reply]
Outside of the scope of core. It's best as a plugin. You can also try space-? to bring up the command palette.

**Reply by notramo** [reply]
When a user first downloads `helix`, they will probably try it without plugins, then, **if they like it**, look for plugins.

I don't see documentation about writing plugins.

**Reply by kirawi** [reply]
There is no plugin system yet. While Helix aims to be batteries-included, this is out-of-scope because it's fairly unnecessary for the vast majority of users who come from similar editors, or who are alright with reading documentation.

**Reply by notramo** [reply]
I think it's not too complicated feature.
I didn't know `helix` is exclusively designed to avoid being first-ever modal editor for new users.

For reference, here is Neovim FTUE page.
![image](https://github.com/helix-editor/helix/assets/26059643/f359a4cd-d50d-4c5b-9be1-0d6a60b00e12)

**Reply by kirawi** [reply]
This was discussed previously, the answer remains no AFAIK. https://github.com/helix-editor/helix/issues/2339

**Comment by notramo** [comment]
See the amazing FTUE of [Nushell](https://www.nushell.sh/) or [Zellij](https://zellij.dev/). I was able to use a quite decent workflow in Zellij without reading any documentation.

**Comment by Abdullah-Mahthel** [comment]
hey guyz please tell me how to set up my Helix on windows im newbi in programming in general 
l like Helix a lot  . How to set path in Environment variable and How can l run a C++ source code  :/
